### PR TITLE
parliament 1.3.1 (new formula)

### PR DESCRIPTION
**Pull request by BastianZim**
parliament 1.3.1 (new formula)

Add formula

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/parliament.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
+class Parliament < Formula
+  include Language::Python::Virtualenv
+
+  desc "AWS IAM linting library"
+  homepage "https://github.com/duo-labs/parliament"
+  url "https://github.com/duo-labs/parliament/archive/1.3.1.tar.gz"
```
Comment (dtrodrigues) [inline Formula/parliament.rb:6]:
```suggestion
  url "https://files.pythonhosted.org/packages/7e/82/26b58070877848c21c59a7c8325117941ec136552425212daa0968e8afcd/parliament-1.3.1.tar.gz"
```
If you change to the PyPI sdist url, `brew bump-formula-pr` will know to update the python resources on a version bump. You can still refer to the source repo by putting 'https://github.com/duo-labs/parliament.git' in the `head` section of the formula.

Comment (BastianZim) [reply Formula/parliament.rb]: Thanks @dtrodrigues, will do!

**Comment by BastianZim** [comment]
@dtrodrigues I'm not 100% sure how to rebase this. Is it just running `git rebase --interactive origin/master` and then selecting the commit at the top?

**Comment by dtrodrigues** [comment]
Yeah - you'll want to rebase and then 'pick' the first commit and 'fixup' the other commits so that you keep their contents but lose their commit messages. The sha and location of the head need to be updated as well.

**Comment by BastianZim** [comment]
@dtrodrigues Would you mind having a look again now? Everything should be incorporated now. Thanks!

**Review by carlocab — APPROVED** [review]
Thanks, @BastianZim.

**Pull request closed without merging**